chainer/function_node.py

```diff
@@ -308,8 +307,6 @@ def apply(self, inputs):
                 self._retained_output_data = tuple(retained_data)
 
             self.lazy_grad_sum = configuration.config.lazy_grad_sum
-            if self.lazy_grad_sum:
-                experimental('config.lazy_grad_sum')
```

Comment:
'config.lazy_grad_sum' is not documented as experimental, but maybe it's ok to just remove experimental status?